### PR TITLE
Added new sections to the landing page

### PR DESCRIPTION
**Pull request by davebcn87**
## Description

Added new sections to the landing page:

* Tweets talking about FlashList
* Case study blog post section
* Expo installation
* Masonry layout
* Conference videos and upcoming conferences/talks

## Testing

To run the website locally you need to run the following commands:

```sh
cd website
bundle exec jekyll serve
```
This will create a localhost site that you can access with your browser using this url: http://127.0.0.1:4000/

### CODE REVIEW

**Comment by ElviraBurchik** [comment]
Love these updates! A few suggestions: 
Screenshot 2022-10-28 at 12 05 04

Screenshot 2022-10-28 at 12 20 58

File: website/index.html [hunk 1]
```diff
@@ -121,21 +124,114 @@ <h3 class="gray">React Native FlatList</h3>
         video=section.video isEven=isEven code=section.code alt=section.alt %}
         {% endfor %}
 
-        <!-- <div class="section casestudy">
+        <div class="section casestudy">
```
[the remark this answers is not in the document]
Comment (davebcn87) [reply website/index.html:127]: We didn't in this case, is that the appropriate channel to ask about that?

**Comment by davebcn87** [comment]
About the tweets, I was thinking on keeping 3 only. I will move the podcast to the other section and rename it.

File: website/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (davebcn87) [reply website/index.html:127]: Ok, waiting for feedback on this.

**Comment by davebcn87** [comment]
@ElviraBurchik addressed your feedback, waiting for the copy review now.

**Review by naqvitalha — COMMENTED** [review]
One suggestion about upcoming events. Otherwise, this looks pretty awesome!

**Comment by davebcn87** [comment]
Already removed the section and asked for copy review. It looks good to be merged. @ElviraBurchik @naqvitalha any concerns?

**Review by naqvitalha — APPROVED** [review]

**Pull request merged**